HappyAnding/HappyAnding/Views/WriteShortcutViews/IconModalView.swift

```diff
@@ -58,10 +58,25 @@ struct IconModalView: View {
     
     var body: some View {
         VStack {
-            Text("아이콘")
-                .Headline()
-                .padding(.top, 16)
-                .padding(.bottom, 24)
+            HStack {
+                Button {
+                    self.isShowingIconModal = true
```

Comment:
```suggestion
                    self.isShowingIconModal = false
```
로 변경해야 닫기 버튼이 동작하네요..!